scriptmodules/emulators/dosbox-staging/speexdsp_simd_disable.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/meson.build b/meson.build
+index 8db4dd4..e5e7549 100644
+--- a/meson.build
++++ b/meson.build
+@@ -396,7 +396,7 @@ endif
+ if not speexdsp_dep.found() or not is_system_speexdsp_reliable
+     speexdsp_dep = subproject(
+         'speexdsp',
+-        default_options: default_wrap_options,
++        default_options: default_wrap_options + ['simd=false']
```

Comment:
Thanks for catching this @joolswills and for flagging it @nemo93.
Nice to see the RPI1 still supported in RetroPie!

For the next Staging release, we'll find a way to check for NEON inside meson.

Reply:
Thanks.